File: Atividades/atividade.003/GestaoEscolar/GestaoEscolar.Web.Api/GestaoEscolar.Web.Api.Service/BoletimService.cs
using GestaoEscolar.Web.Api.Model;
using GestaoEscolar.Web.Api.Repository;

namespace GestaoEscolar.Web.Api.Service
{
    public class BoletimService : Service<AlunoDisciplina>
    {
        public BoletimService(AlunoDisciplinaRepository repository) : base(repository)
        {
            
        }
    }
}

File: Atividades/atividade.005/escola/src/app/core/services/boletim.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';

import { Boletim } from '../models/boletim.model';
import { Service } from './service';

@Injectable({
  providedIn: 'root'
})
export class BoletimService extends Service<Boletim> {
  constructor(httpClient: HttpClient) {
    super(httpClient, 'alunodisciplina');
  }
}
